back/app/models/notifications/cosponsor_of_your_initiative.rb

```diff
@@ -0,0 +1,90 @@
+#  invite_id                     :uuid
+#  reason_code                   :string
+#  other_reason                  :string
+#  post_status_id                :uuid
+#  official_feedback_id          :uuid
+#  phase_id                      :uuid
+#  post_type                     :string
+#  post_status_type              :string
+#  project_folder_id             :uuid
+#  inappropriate_content_flag_id :uuid
+#  internal_comment_id           :uuid
+#  basket_id                     :uuid
+#  cosponsors_initiative_id      :uuid
+#
+# Indexes
+#
+#  index_notifications_on_basket_id                            (basket_id)
+#  index_notifications_on_cosponsors_initiative_id             (cosponsors_initiative_id)
+#  index_notifications_on_created_at                           (created_at)
+#  index_notifications_on_inappropriate_content_flag_id        (inappropriate_content_flag_id)
+#  index_notifications_on_initiating_user_id                   (initiating_user_id)
+#  index_notifications_on_internal_comment_id                  (internal_comment_id)
+#  index_notifications_on_invite_id                            (invite_id)
+#  index_notifications_on_official_feedback_id                 (official_feedback_id)
+#  index_notifications_on_phase_id                             (phase_id)
+#  index_notifications_on_post_id_and_post_type                (post_id,post_type)
+#  index_notifications_on_post_status_id                       (post_status_id)
+#  index_notifications_on_post_status_id_and_post_status_type  (post_status_id,post_status_type)
+#  index_notifications_on_recipient_id                         (recipient_id)
+#  index_notifications_on_recipient_id_and_read_at             (recipient_id,read_at)
+#  index_notifications_on_spam_report_id                       (spam_report_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (basket_id => baskets.id)
+#  fk_rails_...  (comment_id => comments.id)
+#  fk_rails_...  (cosponsors_initiative_id => cosponsors_initiatives.id)
+#  fk_rails_...  (inappropriate_content_flag_id => flag_inappropriate_content_inappropriate_content_flags.id)
+#  fk_rails_...  (initiating_user_id => users.id)
+#  fk_rails_...  (internal_comment_id => internal_comments.id)
+#  fk_rails_...  (invite_id => invites.id)
+#  fk_rails_...  (official_feedback_id => official_feedbacks.id)
+#  fk_rails_...  (phase_id => phases.id)
+#  fk_rails_...  (project_id => projects.id)
+#  fk_rails_...  (recipient_id => users.id)
+#  fk_rails_...  (spam_report_id => spam_reports.id)
+#
+module Notifications
+  class CosponsorOfYourInitiative < Notification
+    validates :initiating_user, :cosponsors_initiative, presence: true
+
+    ACTIVITY_TRIGGERS = { 'CosponsorsInitiative' => { 'cosponsorship_accepted' => true } }
+    EVENT_NAME = 'Cosponsor has accepted your invitation to cosponsor your initiative'
+
+    def self.make_notifications_on(activity)
+      cosponsors_initiative = activity&.item
+      initiating_user = cosponsors_initiative&.user
```

Comment:
Usually, as in your other notification, it's `activity&.user`. Are they the same?
```suggestion
      initiating_user = activity&.user
```

Reply:
Yes, I think they are. I'll change it (and check it still works / update the related test).